wire-library/wire-protoc/src/main/java/com/squareup/wire/protocwire/WireGenerator.kt

```diff
@@ -146,19 +150,68 @@ private fun parseFileDescriptor(fileDescriptor: FileDescriptorProto, descs: Desc
     types.add(parseEnum(messagePath, helper, enumType, descs))
   }
 
+  val services = mutableListOf<ServiceElement>()
+  val servicePath = mutableListOf(FileDescriptorProto.SERVICE_FIELD_NUMBER, 0)
+  for ((index, service) in fileDescriptor.serviceList.withIndex()) {
+    servicePath[servicePath.size - 1] = index
+    services.add(parseService(servicePath, helper, service, descs))
+  }
 
   return ProtoFileElement(
     location = Location.get(fileDescriptor.name),
     imports = imports,
     publicImports = publicImports,
     packageName = if (fileDescriptor.hasPackage()) fileDescriptor.`package` else null,
     types = types,
-    services = emptyList(),
+    services = services,
     options = parseOptions(fileDescriptor.options, descs),
     syntax = Syntax.PROTO_3,
```

Comment:
https://github.com/buildbreaker/wire/pull/17/files#diff-88d5d6d433c5b254f1aeeaef7ef5c753755dfabe06474dcca758639cc56c27aeR171

fixed in this pr